spec/data_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'yaml'
+require 'spec_helper'
+
+RSpec.describe 'data' do
+  it 'has the same keys across all locales' do
+    keys = Dir['_data/*/settings.yml'].map { |path| [path, hash_keys(YAML.load_file(path))] }
+
+    first_file_keys, *other_file_keys = keys
+    first_file, first_keys = first_file_keys
```

Comment:
At the risk of adding yet another dismissed review, one way to save a line and remove an intermediate variable
```suggestion
    (first_file, first_keys), *other_file_keys = first_file_keys
```

Reply:
Haha, yeah, the review dismissal really discourages feedback / iteration 😕 

Fortunately, I think I prefer it as-is over the alternative. The multi-level destructuring feels a little hard to read, and this is a little more consistent with the `_file_keys` pairing between "first" and "other", and the subsequent destructure of block parameters on line 11.